src/main/java/catan/controllers/PlayController.java

```diff
@@ -32,6 +32,17 @@ public void buildRoad(@RequestParam(value = "token", required = false) String to
         playService.buildRoad(user, gameId, edgeId);
     }
 
+    @RequestMapping(value = "build/settlement",
+            method = RequestMethod.POST,
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    public void buildSettlement(@RequestParam(value = "token", required = false) String token,
+                          @RequestParam("gameId") String gameId,
+                          @RequestParam("nodeId") String nodeId) throws AuthenticationException, GameException, PlayException {
```

Comment:
@RequestParam are on different levels, are you using Reformat code operations to normalize all the code in the class according to common rules?